src/files.ts

```diff
@@ -52,6 +52,11 @@ export function* visitMarkdownFiles(root: string): Generator<string> {
 export function* visitFiles(root: string): Generator<string> {
   const visited = new Set<number>();
   const queue: string[] = [(root = normalize(root))];
+  try {
+    visited.add(statSync(join(root, ".observablehq")).ino);
+  } catch {
+    // ignore the .observablehq directory, if it exists
```

Comment:
I noticed that if you have a `.md` file in your `docs/.observablehq/cache`, it was being picked up by the default pages. We don’t want that, so let’s just ignore everything in `.observablehq` under the root.